src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -3162,11 +3165,14 @@ public static void townAdd(Object sender, Town specifiedTown, String[] names) th
 				else
 					town = specifiedTown;
 			}
-
 		} catch (TownyException x) {
 			TownyMessaging.sendErrorMsg(sender, x.getMessage());
 			return;
 		}
+
+		if(TownySettings.isTownBankruptsyEnabled() && town.isBankrupt())
+			throw new TownyException(TownySettings.getLangString("msg_err_siege_besieged_town_cannot_invite"));

```

Comment:
Change to `if (town.isBankrupt())`
Language message should not be something relating to siegewar.

Reply:
Done